.github/workflows/import/index.js

```diff
@@ -163,8 +163,12 @@ function safeguardMetadataImages(dom) {
   }
 }
 
+const map = {}
 async function importUrl(aemPath, importedMedia) {
   const url = new URL(importFrom + aemPath.replace('.md', ''));
+  // AVOID Re-fetching the same resource (e.g. nested circular fragments)
+  if(map[importFrom + aemPath.replace('.md', '')]) return
+  map[importFrom + aemPath.replace('.md', '')] = true
```

Comment:
Although this is a workflow file, I see it still has the ending semicolons, so maybe
```suggestion
  if(map[importFrom + aemPath.replace('.md', '')]) return;
  map[importFrom + aemPath.replace('.md', '')] = true;
```
It might also be worth defining `importFrom + aemPath.replace('.md', '')` as a constant to avoid some repetition.